src/pageObject/CalculatorPage.ts

```diff
@@ -49,4 +50,71 @@ export class CalculatorPage extends BasePage {
         const cost = await this.totalCostLocator.textContent();
         return cost ? cost.trim() : '';
     }
+
```

Comment:
It looks good, but my recommendation is to define methods for implementing complex logic only. For simple cases it would be enough to expose the locator. In other words, no sense to wrap one line actions into methods.